src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -75,13 +78,23 @@
     public static final SlimefunItemStack SCRAP_MACHINE = new SlimefunItemStack(
         "SCRAP_MACHINE",
         Material.BLACK_CONCRETE,
-        "&8Scrap Machine"
+        "&8Scrap Machine",
+        "",
+        "&fProduces &8Scrap &ffrom anything",
+        "",
+        LoreBuilderDynamic.powerBuffer(450),
+        LoreBuilderDynamic.powerPerSecond(10)
     );
 
     public static final SlimefunItemStack MASS_FABRICATOR_MACHINE = new SlimefunItemStack(
         "MASS_FABRICATOR_MACHINE",
         Material.PURPLE_CONCRETE,
-        "&5Mass Fabricator"
+        "&5Mass Fabricator",
+        "",
+        "&fConverts &8Scrap &fto &5UU-Matter",
+        "",
+        LoreBuilderDynamic.powerBuffer(49998),
+        LoreBuilderDynamic.powerPerSecond(16666)
```

Comment:
These values should just be retrieved from Mass Fab class. I see no reason to manually enter them in 2 places.

Reply:
I need some help here, how can this be done? I've tried `MassFabricator.ENERGY_CONSUMPTION`, but this requires `ENERGY_CONSUMPTION` to be public. 

The buffer is under its own method, how do I retrieve that value?

Reply:
Make it public and make the capacity it's own public constant which is used in the method and here.

Reply:
🤦 I think I finally understand what ur trying to say my dum